{{cookiecutter.project_folder}}/requirements-dev.txt

```diff
@@ -1,13 +1,9 @@
 attrs >=17.4
-black ==22.12.0
-bump2version >=1.0.1
-check-manifest >=0.36
-compas_invocations
-doc8
-flake8
+black >=22.12.0
```

Comment:
there is this recent security release `24.xxx` something

Reply:
so should be >=24?

Reply:
i think so